### DISCUSSION TITLE
Get lists by rank

### DISCUSSION THREAD

**Discussion by mariango2** [trakt/trakt-api#438] [Q&A]
Hi,
is it possible to get a user's lists ordered by rank or to get the rank of a list?
I'd like to sync the order of the user's lists between devices and I know there is a method to send a new order to the API that works. But the method to get the lists doesn't seem to sort the list items by rank and the returned lists don't have a rank attribute

**Comment by tysonkerridge** [comment]
I'm assuming you're using `/users/:id/lists`? I believe it's meant to be sorted by rank. Does your response include the sort info and is it set to rank? eg.
```json
"sort_by": "rank",
"sort_how": "asc",
```

**Reply by mariango2** [reply]
Yes, each list item does include that info with "rank" and now that I've tried it on the next day, the lists were returned by rank.
I guess, the lists are just cached for a certain amount of time and I'll just have to wait to get the new order.
